app/graphql/types/chapter.rb

```diff
@@ -0,0 +1,14 @@
+class Types::Chapter < Types::BaseObject
+  implements Types::MediaUnitInterface
```

Comment:
Despite my stupid ass using `EpisodicInterface`, GraphQL apparently doesn't recommend the suffix of `Interface` so this should probably be `Types::MediaUnit`